docassemble/assemblylinewizard/data/questions/assembly_line.yml

```diff
@@ -550,6 +548,97 @@ code: |
   objects_block.objects.gathered=True
   add_objects_block = True
 ---
+code: |
+  # This will add the questions and built-in fields to the screen_order 
+  # list. This will allow the developer to reorder questions without needing
+  # to restart the interview
+  # TODO: do we want to use this to nudge a recommended question order?
+  screen_order.clear()
+  for question in questions:
+    screen_order.append(question)
+  
+  # Note that some built-in fields are not unique screens. This could
+  # mess up the count
+  unique_fields = set()
+  for field in built_in_fields_used:
+    # Don't add the users[0].signature field to this list
+    if field.final_display_var == "users[0].signature":
+      continue
+    if not field.final_display_var in unique_fields:
+      unique_fields.add(field.final_display_var)
+      screen_order.append(field)
+  screen_order.gathered = True
+  set_initial_screen_order = True
+---
+table: screen_order.table
+rows: screen_order
+columns:
+  - Screen: |
+      row_item.question_text if isinstance(row_item, DAQuestion) else row_item.variable_name_guess if isinstance(row_item, DAField) else 'N/A'
+  - Number of fields: |
+      len(row_item.field_list) if isinstance(row_item, DAQuestion) else 'N/A'
+  - Edit: |
+      action_button_html(url_action(row_item.attr_name('edit_question'))) if isinstance(row_item, DAQuestion) else ''
+allow reordering: True
+---
+table: questions[i].edit_table
+rows: screen_order[i].field_list
+columns:
+  - Field: |
+      row_item.final_display_var
+  - Prompt: |
+      row_item.label
```

Comment:
Minor nit: I'd do prompt as the first table column on the left. That way it mirrors how the actual screen looks.